mods/ra/fluent/lua.ftl

```diff
@@ -82,7 +82,7 @@ defeat-allied-forces = Defeat the Allied forces.
 
 ## allies-08ab
 experiment-successful = The experiment is a success!
-chronosphere-experiment-completes-in = Chronosphere experiment completes in { $time }
+chronosphere-experiment-complete-in = Chronosphere experiment complete in { $time }
```

Comment:
I can't agree to this. It just sounds wrong. This way the sentence is missing an "is"/"will be" which is way more "gheto talk" than proper English.